lib/spack/spack/build_systems/meson.py

```diff
@@ -183,6 +184,8 @@ def meson_args(self):
     def meson(self, pkg, spec, prefix):
         """Run ``meson`` in the build directory"""
         options = [os.path.abspath(self.root_mesonlists_dir)]
+        if self.spec["meson"].satisfies("@0.64:"):
+            options.insert(0, "setup")
```

Comment:
Can we construct the list by just appending, or by resetting the list? I think it would be more readable.
```suggestion
        if self.spec["meson"].satisfies("@0.64:"):
            options = ["setup", os.path.abspath(self.root_mesonlists_dir)]
```